lib/kitchen/driver/gce.rb

```diff
@@ -520,8 +524,8 @@ def boot_disk_source_image
         @boot_disk_source ||= image_url
       end
 
-      def image_url
-        return "projects/#{image_project}/global/images/#{image_name}" if image_exist?
+      def image_url(image = image_name)
+        return "projects/#{image_project}/global/images/#{image}" if image_exist?(image)
```

Comment:
This change breaks in the tests because `image_exist?` currently does not accept a parameter.